platform-tests/src/acceptance/http_edge_cases_test.go

```diff
@@ -65,10 +62,8 @@ var _ = Describe("HTTP edge cases", func() {
 			appName = generator.PrefixedRandomName("CATS-APP-HTTP-TESTER-")
 			Expect(cf.Cf(
 				"push", appName,
-				"-b", config.GoBuildpackName,
 				"-p", "../../example-apps/http_tester",
 				"-d", config.AppsDomain,
-				"-c", "./bin/http_tester; sleep 1; echo 'done'",
```

Comment:
I don't think this is picking up the `manifest.yml`. I think the cf CLI always looks in the current directory for that (even when `-p` is specified), so you'll need to add a `"-f", "../../example-apps/http_tester/manifest.yml",` line.